BrowserPlugin/Plugin/src/background.js

```diff
@@ -0,0 +1,34 @@
+import { VectorTile } from '@mapbox/vector-tile';
+import Protobuf from 'pbf';
+
+const requests = [];
+chrome.storage.local.set({ 'requestsLog': requests });
+
+const requestedUrls = [];
+
+
+
+chrome.webRequest.onCompleted.addListener(
+    details => {
+        const url = details.url;
+        if (!requestedUrls.includes(url) &&
+            url.includes('https://functions.yandexcloud.net/d4esngj8sq9q03pu1f39')) {
```

Comment:
The URL must be parametrised. For example, you can put input element in html, from which to take the desired URL